### PR TITLE
Workflow Automated Test - LRQA-67514 | master

### PR DESCRIPTION
**Pull request by AfonsoHSantos**

### CODE REVIEW

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsProcessListView.testcase [hunk 1]
```diff
@@ -146,12 +146,36 @@ definition {
...
+
+		Navigator.openURL();
 
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			panel = "Site Administration",
+			portlet = "Blogs");
+
+		Blogs.addWithWorkflow(
+			entryContent = "Blogs Entry Content",
```
Comment (CalumR23) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsProcessListView.testcase:170]: @AfonsoHSantos Could you please replace this macro with the Blog API?
`JSONBlog.addEntry(
    entryContent = "Blogs Entry Content",
    entryTitle = "Blogs Entry Title");`

Comment (AfonsoHSantos) [reply modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsProcessListView.testcase:350]: fixed

**Comment by CalumR23** [comment]
ci:test:sf

**Comment by CalumR23** [comment]
ci:forward

**Pull request closed without merging**